README.md

```diff
@@ -1,6 +1,6 @@
 
 # Ruby_Morse_Code
-In this project building OOP school library: add Person, Student and Teacher classes
+In this project building TDD-Project: Test Driven Development
```

Comment:
> - [ ] Please remove this section from your Readme file as it does not correspond with the project title. Doing this will make your project to be professional.👍